daml-lf/transaction/src/main/protobuf/com/digitalasset/daml/lf/value.proto

```diff
@@ -97,6 +99,11 @@ message Variant {
     Value value = 3;
 }
 
+message Enum {
+    Identifier enum_id = 1; // optional
+    string value = 2;
```

Comment:
Again, `value` here seems like a bad name. We should settle one one only -- "constructor", "tag", "name", whatever.